apps/labs/posts/scikit-lego-narwhals.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'How Narwhals and scikit-lego came together to achieve dataframe-agnosticism'
+authors: [marco-gorelli]
+published: May 26, 2024
+description: 'And how your library can become dataframe-agnostic too'
+category: [PyData ecosystem]
+featuredImage:
+  src: /posts/scikit-lego-narwhals/scikit_lego_narwhals_handshake.png
+  alt: 'Scikit-lego and Narwhals logos shaking hands'
+hero:
+  imageSrc: /posts/scikit-lego-narwhals/hero.png
+  imageAlt: 'Scikit-lego and Narwhals logos'
+---
+
+[Scikit-lego](https://github.com/koaning/scikit-lego) is a Python project from the [scikit-learn](https://scikit-learn.org/stable/) ecosystem that contributes extra estimators for machine-learning pipelines.
+It's a relatively popular project with over 1000 Github stars and 20,000 downloads a month.
+It's popularity comes from industry as many of the tools that it provides aren't proven to be state-of-the-art,
```

Comment:
startup industry?

I thought state-of-the-art and industry were nearly synonymous, hence my confusion.

Reply:
not sure what you're asking, sorry, could you please clarify?

Reply:
> In data science, it's often the classic not-really-state-of-the-art stuff that delivers value.

I see. Is that uncontroversial? Is it so obvious to your readers that it needs no explaining? Is industry the only (or almost only) place where data science is deriving value from the tried-and-true, boring, not-hyped-up tools and methods?

Reply:
sure, thanks - i've reworded to not mention industry